### DISCUSSION TITLE
Enable SSL in Docker run

### DISCUSSION THREAD

**Discussion by goncatin** [Stirling-Tools/Stirling-PDF#1123] [General]
Hi.

I am using Stirling-PDF over ubuntu 22.04 with docker run. Everything work fine, but when I change the port from 8080 to 443 via -p 443:8080, and with this configuration in settings.yml:

local:
  server:
    port: 443
server:
  port: 443
  ssl:
    key-store: /certs/my-cert.p12
    key-store-password: ${my-cert-password}

Obviusly I change the data of my-cert.p12 and my-cert-password

The docker container stops. If I delete this lines in settings.yml, obviusly I get ERR_SSL_PROTOCOL_ERROR
¿How can I put the SSL certificate?

Edit: Solved.
Change configuration file to

server:
  ssl:
    certificate: /configs/wildcard.crt
    certificate-private-key: /configs/wildcardkey.pem

And now it works

**Comment by Hessenplay** [comment]
Is it possible to use Stirling 
PDF integrated Webserver with SSl and if so how?

**Reply by jobritz** [reply]
I don't know if StirlingPDF has ssl included in its webserver but if you want to use ssl you could use a reverse proxy like [nginx](https://nginx.org/) works fine for me

**Comment by MartinMDS** [comment]
Hello,
We use traefik to deal with https, works fine.
